packages/chart-advisor/src/advisor/index.ts

```diff
@@ -22,6 +22,8 @@ export type ChartAdviseParams = {
   dataProps?: BasicDataPropertyForAdvice[];
   /** data fields to focus, apply in `data` and `dataProps` */
   fields?: string[];
+  /** specify data types for fields to focus, Please notice that this will modify the data you provided. It will convert the data according to fieldTypes  */
+  fieldTypes?: (analyzer.TypeSpecifics | '')[];
```

Comment:
Thx for mention, dataset should export types like `Extra['columnTypes']`.
![image](https://user-images.githubusercontent.com/15177000/225221734-81a17f03-4aaa-488d-874c-896c6abf6060.png)